.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
```

Comment:
just curious, 

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference

here we have `uv`, so can `uv` be used here?